include/core/ib_particle.h

```diff
@@ -115,6 +114,11 @@ class IBParticle
   // iteration.
   Tensor<1, 3> omega_iter;
 
+  Functions::ParsedFunction<dim> *f_velocity;
+  Functions::ParsedFunction<dim> *f_position;
+  Functions::ParsedFunction<dim> *f_omega;
```

Comment:
Make them shared pointers instead of raw pointers. Raw pointers are very prone to mem leak. We avoid them (I don't think there are any left in Lethe)